### PR TITLE
PHOENIX-6449 Cannot start minicluster in github CI check for connectors

### PR DESCRIPTION
**Pull request by stoty**

### CODE REVIEW

**Comment by stoty** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 29s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  yamllint  |   0m  1s |  There were no new yamllint issues.  |
||| _ Other Tests _ |
| -1 :x: |  asflicense  |   0m 11s |  The patch generated 1 ASF License warnings.  |
|  |   |   2m  3s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-Connectors-PreCommit-GitHub-PR/job/PR-51/1/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/phoenix-connectors/pull/51 |
| Optional Tests | dupname asflicense yamllint |
| uname | Linux 97d1f883df8e 4.15.0-101-generic #102-Ubuntu SMP Mon May 11 10:07:26 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev/phoenix-connectors-personality.sh |
| git revision | master / e6f5512 |
| asflicense | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-Connectors-PreCommit-GitHub-PR/job/PR-51/1/artifact/yetus-general-check/output/patch-asflicense-problems.txt |
| Max. process+thread count | 42 (vs. ulimit of 30000) |
| modules | C: . U: . |
| Console output | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-Connectors-PreCommit-GitHub-PR/job/PR-51/1/console |
| versions | git=2.7.4 maven=3.3.9 yamllint=1.20.0 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by stoty** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 21s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  1s |  No case conflicting files found.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
||| _ Patch Compile Tests _ |
| -1 :x: |  whitespace  |   0m  0s |  The patch has 2 line(s) that end in whitespace. Use git apply --whitespace=fix <<patch_file>>. Refer https://git-scm.com/docs/git-apply  |
| -1 :x: |  yamllint  |   0m  1s |  The patch generated 7 new + 4 unchanged - 0 fixed = 11 total (was 4)  |
||| _ Other Tests _ |
| -1 :x: |  asflicense  |   0m 16s |  The patch generated 1 ASF License warnings.  |
|  |   |   2m  0s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-Connectors-PreCommit-GitHub-PR/job/PR-51/2/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/phoenix-connectors/pull/51 |
| Optional Tests | dupname asflicense yamllint |
| uname | Linux c12552a11198 4.15.0-101-generic #102-Ubuntu SMP Mon May 11 10:07:26 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev/phoenix-connectors-personality.sh |
| git revision | master / e6f5512 |
| whitespace | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-Connectors-PreCommit-GitHub-PR/job/PR-51/2/artifact/yetus-general-check/output/whitespace-eol.txt |
| yamllint | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-Connectors-PreCommit-GitHub-PR/job/PR-51/2/artifact/yetus-general-check/output/diff-patch-yamllint.txt |
| asflicense | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-Connectors-PreCommit-GitHub-PR/job/PR-51/2/artifact/yetus-general-check/output/patch-asflicense-problems.txt |
| Max. process+thread count | 42 (vs. ulimit of 30000) |
| modules | C: . U: . |
| Console output | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-Connectors-PreCommit-GitHub-PR/job/PR-51/2/console |
| versions | git=2.7.4 maven=3.3.9 yamllint=1.20.0 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by stoty** [comment]
The check fails, because PHOENIX-6443 is also needed.
We have successful run in #51 , which has both fixes.

File: .github/actions/tune-runner-vm/action.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: Tune Runner VM
+description: |
+  Fine tunes the GitHub Runner VM operation system for Trino builds.
```
Comment (richardantal) [inline .github/actions/tune-runner-vm/action.yml:3]: I think we should replace Trino here to Phoenix-Connectors

**Review by richardantal — COMMENTED** [review]
It looks good other than the small nit I wrote.
I see it already works on https://github.com/apache/phoenix-connectors/pull/50 thats great.

**Comment by stoty** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   2m  2s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
||| _ Patch Compile Tests _ |
| -1 :x: |  whitespace  |   0m  0s |  The patch has 2 line(s) that end in whitespace. Use git apply --whitespace=fix <<patch_file>>. Refer https://git-scm.com/docs/git-apply  |
| -1 :x: |  yamllint  |   0m  1s |  The patch generated 7 new + 4 unchanged - 0 fixed = 11 total (was 4)  |
||| _ Other Tests _ |
| -1 :x: |  asflicense  |   0m 10s |  The patch generated 1 ASF License warnings.  |
|  |   |   2m 34s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-Connectors-PreCommit-GitHub-PR/job/PR-51/3/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/phoenix-connectors/pull/51 |
| Optional Tests | dupname asflicense yamllint |
| uname | Linux 4fb32688c652 4.15.0-101-generic #102-Ubuntu SMP Mon May 11 10:07:26 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev/phoenix-connectors-personality.sh |
| git revision | master / e6f5512 |
| whitespace | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-Connectors-PreCommit-GitHub-PR/job/PR-51/3/artifact/yetus-general-check/output/whitespace-eol.txt |
| yamllint | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-Connectors-PreCommit-GitHub-PR/job/PR-51/3/artifact/yetus-general-check/output/diff-patch-yamllint.txt |
| asflicense | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-Connectors-PreCommit-GitHub-PR/job/PR-51/3/artifact/yetus-general-check/output/patch-asflicense-problems.txt |
| Max. process+thread count | 46 (vs. ulimit of 30000) |
| modules | C: . U: . |
| Console output | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-Connectors-PreCommit-GitHub-PR/job/PR-51/3/console |
| versions | git=2.7.4 maven=3.3.9 yamllint=1.20.0 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

File: .github/actions/tune-runner-vm/action.yml (hunk 1 above)
Comment (stoty) [reply .github/actions/tune-runner-vm/action.yml]: Thank you.
I have removed the mention of Trino.

**Comment by richardantal** [comment]
LGTM +1

**Pull request closed without merging**